app/controllers/api/v1/release_engines/oci/tags_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Api::V1::ReleaseEngines
+  class Oci::TagsController < Api::V1::BaseController
+    before_action :require_ee!
+    before_action :scope_to_current_account!
+    before_action :require_active_subscription!
+    before_action :authenticate_with_token
+    before_action :set_package
+
+    def index
+      authorize! package, to: :show?
+
+      releases = authorized_scope(package.releases)
+      authorize! releases
+
+      tags = releases.pluck(:tag)
```

Comment:
Needs to be in lexical order (i.e. case-insensitive alphanumeric order) per the spec.